io.catenax.shared.part_site_information_as_built/1.0.0/PartSiteInformationAsBuilt.ttl

```diff
@@ -0,0 +1,85 @@
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.shared.part_site_information_as_built:1.0.0#>.
+@prefix ext-uuid: <urn:samm:io.catenax.shared.uuid:1.0.0#>.
+
+:PartSiteInformationAsBuilt a samm:Aspect;
+   samm:preferredName "Part Site Information As Built"@en;
+   samm:description "The aspect provides site related information for a given item(i.e. a part type or part instance that is uniquely identifiable within Catena-X via its Catena-X ID). A site is a delimited geographical area where a legal entity does business. In this context all potentially related sites are listed including all sites where e.g. production of this part(type)."@en;
+   samm:properties(:sites :catenaXId);
+   samm:operations();
+   samm:events().
+
+:sites a samm:Property;
+   samm:preferredName "Sites"@en;
+   samm:description "A site is a delimited geographical area where a legal entity does business(geographical address with geo coordinates).A site always has a primary physical address. It is possible that further physical addresses are specified for the site. P.O. box details are only possible in addition to the physical address. A site has a 1:n relation to addresses, means at least 1 address is necessary and multiple addresses are possible."@en;
+   samm:see <https://confluence.catena-x.net/x/QkBHAw>;
+   samm:characteristic :SitesCharacteristic.
+
+:catenaXId a samm:Property;
+   samm:preferredName "Catena-X Identifier"@en;
+   samm:description "The Catena-X ID of the given item(i.e. a part type or part instance), valid for the Catena-X dataspace."@en;
+   samm:characteristic ext-uuid:UuidV4Trait;
+   samm:exampleValue "580d3adf-1981-44a0-a214-13d6ceed9379".
+
+:SitesCharacteristic a samm-c:Set;
+   samm:preferredName "Sites Characteristic"@en;
+   samm:description "Sites Characteristic"@en;
+   samm:dataType :SitesEntity.
+
+:SitesEntity a samm:Entity;
+   samm:preferredName "Site Entity"@en;
+   samm:description "Site Entity"@en;
+   samm:properties(:catenaXsiteId :function).
+
+:catenaXsiteId a samm:Property;
+   samm:preferredName "Catena-X site identifier"@en;
+   samm:description "The identifier of the site according to Catena-X BPDM. The catenaXsiteId must be a valid Catena-X BPN. The BPN is a unique, unchangeable identifier for Business Partners / company locations from foundation to closure, regardless of the different business relationships / structures between or within the Business Partners or company locations."@en;
+   samm:characteristic :BpnCharacteristic;
```

Comment:
You can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR104.
Just chose one correct for you from different type of BPN types in shared model.